src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -11,24 +11,37 @@
 import seedu.address.model.Model;
 import seedu.address.model.pdf.Pdf;
 
+import java.io.File;
+import java.nio.file.Paths;
+
 /**
  * Deletes a pdf identified using it's displayed index from the pdf book.
  */
 public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
 
+    public enum DELETE_TYPE {
+        HARD, SOFT
+    }
+
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the PDF identified by the index number used in the displayed PDF list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Hard or soft delete can be chosen (soft delete selected by default).\n"
+            + "Parameters(Soft Delete): INDEX (must be a positive integer)\n"
+            + "Parameters(Hard Delete): INDEX (must be a positive integer) hard\n"
+            + "Example: " + COMMAND_WORD + " 1 | " + COMMAND_WORD + " 1 hard";
```

Comment:
And I think maybe can remove the " | " cause like it might be confusing for a non-technical user. We could just put "OR" instead?